Cql/Cql.Packaging/ResourcePackager.cs

```diff
@@ -29,8 +29,9 @@ public class ResourcePackager
         /// </summary>
         /// <param name="elmDir">directory to find the ELM files</param>
         /// <param name="cqlDir">directory to find the CQL files</param>
+        /// <param name="resourceCanonicalRootUrl">root part of the resource canonical; otherwise just a '#'</param>
         /// <param name="afterPackageMutator">optional mutator for the resources prior to writing</param>
-        public void Package(DirectoryInfo elmDir, DirectoryInfo cqlDir, Action<IEnumerable<Resource>>? afterPackageMutator = null)
+        public void Package(DirectoryInfo elmDir, DirectoryInfo cqlDir, string? resourceCanonicalRootUrl = null, Action<IEnumerable<Resource>>? afterPackageMutator = null)
```

Comment:
This change would cause a compiler error for current callers, right?  Could you place the new argument last?

Reply:
Maybe it's a good time now to extract these parameters into a class and create an overload for it while keeping the existing method?